core/dbt/contracts/graph/nodes.py

```diff
@@ -1441,6 +1441,12 @@ def same_contents(self, old: Optional["Exposure"]) -> bool:
     def group(self):
         return None
 
+    def __post_serialize__(self, dct: Dict, context: Optional[Dict] = None):
+        dct = super().__post_serialize__(dct, context)
+        if "_event_status" in dct:
```

Comment:
why is this necessary?

Reply:
You're right, this field is never written out.
Fixed the failing test to account for this field instead.

Reply:
no sweat, just reverted the commit here